third_party/js.bzl

```diff
@@ -17,437 +17,473 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@io_bazel_rules_closure//closure:defs.bzl", "filegroup_external")
```

Comment:
Please consider removing all whitespace changes (even if it is ill-formatted) for sake of smaller diff

Reply:
Done. Sorry about that.